cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -178,7 +185,7 @@ <h2>
                 - No min and max but a median: "29.74% median purchase APR"
                 - Single APR for everyone: "24.90% purchase APR"
             #}
-            <div class="h2 u-mb0">
+            <div class="h2 u-mb0" role="definition" id="card-purchase-apr">
```

Comment:
> `<p><span class="…">9.90%</span> <span class="…">Purchase APR</span></p>`

I love this suggestion! It fixes the issue with screenreaders breaking up the text because they're separated as block level elements.